test/rules.jl

```diff
@@ -31,8 +31,42 @@ _second(t) = Base.tuple_type_head(Base.tuple_type_tail(t))
 
     frx, fr = frule(cool, 1)
     @test frx == 2
-    @test fr(1) == 1
-    rrx, rr = rrule(cool, 1)
+    @test fr(NamedTuple(), 1) == (1,)
+    rrx, (rr) = rrule(cool, 1)
```

Comment:
why the brackets around `rr`?

Reply:
Left over from when things were different

```suggestion
    rrx, rr = rrule(cool, 1)
```